pre-docs/bolt_configuration_options.md

```diff
@@ -225,3 +225,23 @@ The `puppetfile` section configures how modules are retrieved when running `bolt
 `proxy`: The HTTP proxy to use for Git and Puppet Forge operations
 
 `forge`: A subsection which can have its own `proxy` setting to set an HTTP proxy for only Puppet Forge operations, and a `baseurl` setting to specify a different Forge host
+
+## Plugin hooks configuration options
```

Comment:
maybe mention that `puppet_library` is the only configurable hook now though we'll likely to forget to keep that up to date.

I think most of the specifics about how this hook works should be in Install Libaries section.